src/dvc_render/vega_templates.py

```diff
@@ -610,15 +492,53 @@ class LinearTemplate(Template):
                 "title": Template.anchor("y_label"),
                 "scale": {"zero": False},
             },
-            "color": {"field": "rev", "type": "nominal"},
+            "color": {
+                "field": "rev",
+                "type": "nominal",
+                "legend": {"symbolOpacity": 1},
+            },
         },
+        "params": [
+            {
+                "name": "opacity",
```

Comment:
tbh, it's not very obvious from the user perspective - why would I care about points opacity?

(toggle on off was more clear to me, ideally we would avoid that one as well of course ... by at least it makes sense it we can't solve the underlying issue)

Reply:
hmm, yes, opacity is common (the one that comes to my mind is a transparent outline of an original curve when smooth is applied), but I would expect it to be applied per line (to see other lines)... or at least that applies to the whole line, not only points

points opacity sounds very specific to the problem, doesn't give same level of benefit as smooth setting

> and it's a common way to deal with too many points.

could you give some examples?